core/src/test/scala/com/microsoft/azure/synapse/ml/nbtest/DatabricksUtilities.scala

```diff
@@ -29,10 +29,11 @@ object DatabricksUtilities {
 
   // ADB Info
   val Region = "eastus"
-  val PoolName = "synapseml-build-10.4"
-  val GpuPoolName = "synapseml-build-10.4-gpu"
-  val AdbRuntime = "10.4.x-scala2.12"
-  val AdbGpuRuntime = "10.4.x-gpu-ml-scala2.12"
+  val PoolName = "synapseml-build-13.3"
+  val GpuPoolName = "synapseml-build-13.3-gpu"
+  val AdbRuntime = "13.3.x-scala2.12"
+  // https://docs.databricks.com/en/release-notes/runtime/13.3lts-ml.html
```

Comment:
nit: is this necessary?

Reply:
We have the release note link on the master for Runtime 11.2, so maintained the same and pointed to Spark34 runtime